tests/apps/testutils.py

```diff
@@ -46,7 +47,7 @@ def start(self, wait_time):
         print(f"running new Docker container with image {image.id}")
         self.container = self.client.containers.run(
             image.id, cmd,
-            network_mode=DOCKER_NETWORK_MODE, volumes=[f"{self.config_dir}:/{CONFIG}"], detach=True)
+            network_mode=DOCKER_NETWORK_MODE, volumes=[f"{self.config_dir}:/{CONFIG}"], detach=True, ports=DOCKER_PORTS)
```

Comment:
note that some tests start hosts on other ports also--you can use `flags.get("http_port", DEFAULT_PORT)` to see which port to map